build/yaml/FunctionalTestsBotsbuild.yml

```diff
@@ -11,20 +11,211 @@ steps:
     version: 3.1.x
 
 - task: NuGetToolInstaller@1
+  displayName: 'Use NuGet'
+
+# Start of Restore & Build excluding Composer and V3
+- task: DotNetCoreCLI@2
+  displayName: Restore bots
   inputs:
-    versionSpec: 
+    command: restore
+    projects: |
+      Bots/DotNet/**/*.csproj
+      !Bots/DotNet/**/Composer/**/*.csproj
+      !Bots/DotNet/Skills/CodeFirst/EchoSkillBot-v3/EchoSkillBot-v3.csproj
+    feedsToUse: 'select'
+    includeNuGetOrg: true
 
-- task: NuGetCommand@2
+- task: DotNetCoreCLI@2
+  displayName: Build bots
   inputs:
-    command: 'restore'
-    restoreSolution: 'Bots/DotNet/FunctionalTestsBots.sln'
+    command: build
```

Comment:
>command: build [](start = 4, length = 14)

Is this doing a restore again?

Reply:
Hi @mrivera-ms  Yes, it was doing the restore in both Restore and Build tasks.
We added the necessary arguments to 'no build' and 'no restore' in the Restore and Build task respectively. Thanks!